server/provider/networkProvider_test.go

```diff
@@ -20,7 +20,7 @@ func TestNewNetworkProvider(t *testing.T) {
 		ObservedProjectedShardIsSet: true,
 		ObserverUrl:                 "http://my-observer:8080",
 		ObserverPubkey:              "abba",
-		ChainID:                     "T",
+		ChainID:                     "devnet",
```

Comment:
The old value was more relevant, testing wise

Reply:
Actually I've made a mistake - we need to have two separate concepts: chain ID and chain name. I will repair this tomorrow.